include/solvers/physical_properties_manager.h

```diff
@@ -23,10 +23,29 @@
 #include <core/rheological_model.h>
 #include <core/specific_heat_model.h>
 #include <core/thermal_conductivity_model.h>
+#include <core/thermal_expansion_model.h>
+#include <core/tracer_diffusivity_model.h>
 
 using namespace dealii;
 
 
+DeclException1(
+  RequiresConstantDensity,
+  std::string,
+  "The following assembler or post-processing utility "
+    << arg1
+    << " that you are trying to use does not support the use of a non-constant density model. Modifications to Lethe are required to take this into account.");
```

Comment:
why is this line to long? It should have been cut by the clang formatting, no?

Reply:
Good question. I'll manually break it into multiple lines.